### ISSUE TITLE
Doesn't understand escaped apostrophe

### ISSUE THREAD

**Issue by tomekwi** [jakubpawlowicz/clean-css#462]
This works:

```css
.a-simple-class {
  some: stuff;
}

.another-one {
  other: stuff;
}
```
```sh
$ cleancss < this
.a-simple-class{some:stuff}.another-one{other:stuff}
```

That fails:

```css
.a-simple-class {
  some: stuff;
}

.this-class\'s-got-an-apostrophe {
  other: stuff;
}

.this-will-not-be-minified-as-well {
  other: stuff;
}
```
```sh
$ cleancss < that
.a-simple-class{some:stuff}

.this-class\'s-got-an-apostrophe {
  other: stuff;
}

.this-will-not-be-minified-as-well {
  other: stuff;
}
```

More about escaped characters in [this article by Mathias Bynens][1].
[1]: https://mathiasbynens.be/notes/css-escapes "CSS character escape sequences"

**Comment by jakubpawlowicz** [comment]
It's a bug indeed. Good catch :+1:

**Comment by jakubpawlowicz** [comment]
Fixed in 3.1.2.

**Comment by tomekwi** [comment]
Thanks! :+1:
